tests/calculations/test_pressure.py

```diff
@@ -0,0 +1,216 @@
+"""Tests for pressure functionality in geometry optimisation calculations."""
+
+from __future__ import annotations
+
+from aiida.engine import run
+from aiida.orm import Bool, Dict, Float, Int, Str, StructureData
+from aiida.plugins import CalculationFactory
+from ase.build import bulk
+from ase.io import write
+from click.testing import CliRunner
+
+from aiida_mlip.data.model import ModelData
+
+
+# Structure changes depending on pressure applied
+def test_pressure_optimization(model_folder, janus_code):
+    """Test geometry optimization with external pressure applied."""
+    model_file = model_folder / "mace_mp_small.model"
+
+    # Test with different pressure values
+    pressure_values = [0.0, 5.0, 10.0]
+    results = {}
+
+    for pressure in pressure_values:
+        inputs = {
+            "metadata": {"options": {"resources": {"num_machines": 1}}},
+            "code": janus_code,
+            "arch": Str("mace"),
+            "struct": StructureData(ase=bulk("NaCl", "rocksalt", 5.63)),
+            "model": ModelData.from_local(model_file, architecture="mace"),
+            "device": Str("cpu"),
+            "opt_cell_fully": Bool(True),
+            "fmax": Float(0.1),
+            "steps": Int(1000),
+            "pressure": Float(pressure),
+            "minimize_kwargs": Dict(
+                {"traj_kwargs": {"filename": f"traj-{pressure}GPa.xyz"}}
+            ),
```

Comment:
We probably don't need this. All it's really doing is checking we change the filename, which we do elsewhere.